### DISCUSSION TITLE
Export Cue to Yaml

### DISCUSSION THREAD

**Discussion by roylisto** [cuelang/cue#1046] [Q&A] [answered]
Hi All, 
I have a cue file and want to export it to yaml, 
I can export it by this command  `cue export filename.cue -o "filename.yaml"`.
My question is this the proper way to export to yaml? because I cannot find it in the docs.

**Comment by myitcv** [comment, marked as answer]
> My question is this the proper way to export to yaml? because I cannot find it in the docs.

Yes, `cue export` is the correct way. And you are right to point out an omission in the docs:

https://cuelang.org/docs/integrations/yaml/

Contrast the JSON section:

https://cuelang.org/docs/integrations/json/#generate-json

I have raised https://github.com/cuelang/cue/issues/1048 to fix this omission.

**Reply by roylisto** [reply]
Okay @myitcv , thank you for your answer

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/1046.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
